apps/web/next.config.js

```diff
@@ -381,6 +390,35 @@ const nextConfig = {
           },
         ],
       },
+      ...[
```

Comment:
Added response header `X-Cal-Org-path` to easily identify to which path the rewrite occured. It would also help in writing checkly tests.